developer/src/kmc-kmn/test/fixtures/000.kmn

```diff
@@ -0,0 +1,11 @@
+c Description: Tests null keyboard 
```

Comment:
I noticed on #8529 you called this `k_000___null_keyboard.kmn`.

Is the plan to consolidate using common/test/keyboards/?

Reply:
Good catch, will do this in a follow-up PR